### ISSUE TITLE
Possible bug caused by inconsistent bilinear interpolation

### ISSUE THREAD

**Issue by AtlantixJJ** [mseitzer/pytorch-fid#14]
Hi, I know you are looking for a bug of why pytorch fid behaves different with tf fid, and I am also searching for why.

I found that the bilinear interpolation output is different of pytorch (1.1) and tensorflow (1.13.1). See this issue (https://github.com/pytorch/pytorch/issues/10604).

Hope it helps. b.t.w. could you tell me how different is FID in this repo with that of original? It is quite necessary for me. Thank you!

**Comment by AtlantixJJ** [comment]
Here is an example:

```python
import torch
import tensorflow as tf
import numpy as np

a = np.random.rand(1, 1, 5, 5).astype("float32")
x_torch = torch.from_numpy(a)
x_tf = a.transpose(0, 2, 3, 1)

x = tf.placeholder(tf.float32, [None, 5, 5, 1])

config = tf.ConfigProto()
config.gpu_options.allow_growth = True
sess = tf.Session(config=config)

y_tf = sess.run(tf.image.resize_bilinear(x, (7, 7)), {x:x_tf})[0, :, :, 0]
y_torch = torch.nn.functional.interpolate(x_torch, (7, 7)).detach().cpu().numpy()[0, 0, :, :]
print(y_tf, y_torch)
```

**Comment by mseitzer** [comment]
Hi, thanks for the hint. I will take a look at this when I get the time. 

It turns out that the Inception model of Pytorch uses different weights and a slightly different model than Tensorflow's Inception. This introduced a significant source of error. I pushed an updated version of the model (see #16), which fixes the errors. 

I do not have general numbers for how different FID scores are between TF and Pytorch. In one of my experiments (after the fix), the results were very similar (`0.08` error), which I think is decent. Of course, this might be different for your particular case. Feel free to investigate and report some numbers.
